Hazelcast.Net/Hazelcast.Client.Spi/ClientPartitionService.cs

```diff
@@ -190,7 +190,7 @@ private void GetPartitionsBlocking()
         {
             while (!GetPartitions() && _live.Get())
             {
-                Thread.Sleep(PartitionRefreshPeriod);
+                Thread.Sleep(100);
```

Comment:
Could we use some meaningful constant name here instead of the value?

Reply:
Can you do it on master 3.12 branch

Reply:
ok.